testsuite/MDAnalysisTests/parallelism/test_multiprocessing.py

```diff
@@ -125,6 +127,23 @@ def test_universe_unpickle_in_new_process():
     assert_equal(ref, res)
 
 
+def create_universe():
+    return mda.Universe(GRO, XTC)
+
+
+def test_creating_multiple_universe_without_offset():
```

Comment:
The tested XTC file is way too short to induce a race scenario here (at least on my computer).

Reply:
Is this test useful if it does not normally trigger the problem?